https/gifts.js

```diff
@@ -5,113 +5,56 @@ also has functions to return admin stuff as well
 */
 const functions = require('firebase-functions');
 require('firebase/firestore');
-const { getUUID, getGeneralQueries } = require('../utils/utils');
-const { YEAR } = require("../config/constants");
+const { getUUID, getGeneralQueries, markGifteeAccount } = require('../utils/utils');
+const { EVENT } = require("../config/constants");
 const db = require('../config/db');
 
 // marks gift as went
-const sendGift = functions.https.onCall(async({user}, context) => {
-  // marks their own account
+const sendGift = functions.https.onCall(async({user, value, giftee_uuid}, context) => {
+  // this has to mark both the giftee and gifter
 
+  if(typeof value === "undefined"){value = true}
+  // gifter first
   let uuid = await getUUID(user)
   if(uuid.error){return {error: "no API key set"}}
-  uuid = uuid.success
+  let entryResult = await db.collection('events').doc(EVENT).collection('participants').doc(uuid.success).set({ sent_own: value }, {merge: true}).then(()=> {return true}).catch(() => {return false});
```

Comment:
We need to set up a linter so long lines like this can die 😄

Reply:
Whats wrong with long lines? The only thing that is off is from .then but the rest of it is literally inserting a document

Reply:
It may be a personal preference, but I find it easier to read when sequential functions are split into multiple lines. Just nitpicking!